### PR TITLE
golangci-lint: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/golangci-lint.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# golangci-lint
+
+> Corredor de linters Go paralelizado, inteligente y rápido que se integra con los principales IDEs y soporta configuración YAML.
+> Más información: <https://golangci-lint.run/usage/quick-start/>.
+
+- Ejecuta linters en la carpeta actual:
+
+`golangci-lint run`
+
+- Lista de linters habilitados y deshabilitados (Nota: los linters deshabilitados se muestran en último lugar, no los confundas con los habilitados):
```
Comment (tricantivu) [inline pages.es/common/golangci-lint.md:10]:
```suggestion
- Lista los linters habilitados y deshabilitados (Nota: los linters deshabilitados se muestran en último lugar, no los confundas con los habilitados):
```

Comment (tricantivu) [inline pages.es/common/golangci-lint.md:10]:
```suggestion
- Lista de linters habilitados y deshabilitados (Nota: los linters deshabilitados se muestran en el último lugar, no los confundas con los habilitados):
```

File: pages.es/common/golangci-lint.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# golangci-lint
+
+> Corredor de linters Go paralelizado, inteligente y rápido que se integra con los principales IDEs y soporta configuración YAML.
```
Comment (tricantivu) [inline pages.es/common/golangci-lint.md:3]:
```suggestion
> Corredor de linters Go paralelizado, inteligente y rápido que se integra con los principales entornos de desarrollo integrado y soporta configuración en YAML.
```

File: pages.es/common/golangci-lint.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# golangci-lint
+
+> Corredor de linters Go paralelizado, inteligente y rápido que se integra con los principales entornos de desarrollo integrado y soporta configuración en YAML.
+> Más información: <https://golangci-lint.run/usage/quick-start/>.
+
+- Ejecuta linters en la carpeta actual:
+
+`golangci-lint run`
+
+- Lista los linters habilitados y deshabilitados (Nota: los linters deshabilitados se muestran en último lugar, no los confundas con los habilitados):
```
Comment (kbdharun) [inline pages.es/common/golangci-lint.md:10]: @tricantivu's suggestion seem to be marked as outdated, so created a new one for the same.
```suggestion
- Lista los linters habilitados y deshabilitados (Nota: los linters deshabilitados se muestran en el último lugar, no los confundas con los habilitados):
```

**Review by kbdharun — APPROVED** [review]

**Review by tricantivu — APPROVED** [review]

**Pull request merged**